user/enterprise/operations-manual.md

```diff
@@ -83,9 +83,14 @@ $ sudo restart travis-worker
 #### Ports are not open Security groups / firewall
 
 A source for the problem could be that the worker machine is not able to communicate with the platform machine.
+
 Here we're distinguishing between an AWS EC2 installation and an installation running on other hardware. For the former, security groups need to be configured per machine. To do so, please follow our installation instructions [here](/user/enterprise/installation/#11-Create-a-Security-Group). If you're not using AWS EC2, please make sure that the ports listed [in the docs](/user/enterprise/installation/#11-Create-a-Security-Group) are open in your firewall.
 
-If none of the steps above lead to results for you, please follow the steps in [#Contact-support](#Contact-support) to move forward.
+#### Docker Versions Mismatched 
+
+Upon inspecting the `travis-worker.log` file, you may notice an error: `Error response from daemon:client and server don't have same version`, which is especially common on workers installed before November 2017 or systems running a `docker version` before `17.06.2-ce`. For this issue, we recommend [re-installing worker from scratch](/user/enterprise/installation/#2.1.-Install-Travis-CI-Worker-on-the-second-host) on a fresh instance. Please note: the default build environment images will be pulled and you may need to apply customizations again as well.
```

Comment:
Do we mentioned before, where exactly `travis-worker.log` can be found? If not, let's use the absolute path here.

Reply:
we do elsewhere, but agreed - might as well include the absolute path here (better idea anyway)